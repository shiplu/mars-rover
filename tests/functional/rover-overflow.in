5 3
1 1 N
MM
1 2 N
MM
1 3 N
MM
2 1 N
MM
2 2 N
MM
2 3 N
MM
3 1 N
MM
3 2 N
MM
3 3 N
MM
4 1 N
MM
4 2 N
MM
4 3 N
MM
5 1 N
MM
5 2 N
MM
5 3 N
MM
