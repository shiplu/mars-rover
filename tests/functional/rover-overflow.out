1 1 N STOPPED
1 2 N STOPPED
1 3 N STOPPED
2 1 N STOPPED
2 2 N STOPPED
2 3 N STOPPED
3 1 N STOPPED
3 2 N STOPPED
3 3 N STOPPED
4 1 N STOPPED
4 2 N STOPPED
4 3 N STOPPED
5 1 N STOPPED
5 2 N STOPPED
5 3 N STOPPED
